src/locales/en/translation.json

```diff
@@ -98,6 +98,30 @@
     "invited": "invited",
     "deactivated": "deactivated"
   },
+  "addMember": {
+    "title": "Invite new member",
+    "fields": {
+      "firstName": {
+        "label": "Name"
+      },
+      "lastName": {
+        "label": "Surname"
+      },
+      "email": {
+        "label": "E-mail"
+      }
+    },
+    "formRequiredError": "Please, fill-in the required fields",
+    "buttons": {
+      "back": "Back",
+      "submit": "Invite"
+    },
+    "success": {
+      "btnHome": "Home",
+      "message": "Member invited",
+      "btnAdd": "Add yet member"
+    }
+  },
```

Comment:
Also

line 6: "transaction": "Transactions"
line 161: "formRequiredError": "Please, fill-in required fields"

Reply:
done